conans/test/functional/editable/commands/inspect_test.py

```diff
@@ -0,0 +1,59 @@
+# coding=utf-8
+
+import os
+import textwrap
+import unittest
+
+from conans.model.ref import ConanFileReference
+from conans.paths.package_layouts.package_editable_layout import CONAN_PACKAGE_LAYOUT_FILE
+from conans.test.utils.tools import TestClient
+
+
+class InspectCommandTest(unittest.TestCase):
+    conanfile_base = textwrap.dedent("""\
+        from conans import ConanFile
+
+        class APck(ConanFile):
+            {body}
+        """)
+    conanfile = conanfile_base.format(body="pass")
+
+    conan_package_layout = textwrap.dedent("""\
+        [includedirs]
+        src/include
+        """)
+
+    def setUp(self):
+        self.ref_parent = ConanFileReference.loads("parent/version@user/name")
+        self.reference = ConanFileReference.loads('lib/version@user/name')
+
+        self.t = TestClient()
+        self.t.save(files={'conanfile.py': self.conanfile})
+        self.t.run('create . {}'.format(self.ref_parent))
+
+        self.t.save(files={'conanfile.py':
+                          self.conanfile_base.format(body='requires = "{}"'.format(self.ref_parent)),
+                      CONAN_PACKAGE_LAYOUT_FILE: self.conan_package_layout, })
+        self.t.run('link . {}'.format(self.reference))
+        self.assertTrue(self.t.client_cache.installed_as_editable(self.reference))
+
+    def tearDown(self):
+        self.t.run('link {} --remove'.format(self.reference))
+        self.assertFalse(self.t.client_cache.installed_as_editable(self.reference))
+        self.assertFalse(os.listdir(self.t.client_cache.conan(self.reference)))
+
+    def test_local(self):
+        self.t.run('inspect .')
```

Comment:
This test has nothing to do with editables. It is like the most basic "inspect" test. Am I missing something? Why is this test here and what is testing?

Reply:
It was here just for completeness (aka copy/paste), all commands in the other tests are running the `conan <command> .` and the `conan <command> <reference>`. I'm removing this one now.